packages/components/atoms/f-card/test-utils/component-objects/f-card.component.js

```diff
@@ -1,7 +1,9 @@
-const Page = require('@justeat/f-wdio-utils/src/base.page');
+import Page from '@justeat/f-wdio-utils';
 
-module.exports = class Card extends Page {
+class Card extends Page {
     constructor () {
         super('atom', 'card-component');
     }
-};
+}
+
+export default new Card();
```

Comment:
This is a bit unusual, what's the benefit of this over just `export default Card;`?

Reply:
Well, naturally it means you don't have to create a new instance of the Card whenever you import it... I just haven't seen it done like this before 😄

Reply:
As long as we don't store any state between tests, it would be okay, but IMO I think it would be fine to `new` up a `Card` for each test just to prevent any leaking between tests in future

Reply:
Yeah, no state is shared across tests, so we'll be fine. Was just following the Page Object docs on WDIO's site and they follow this same structure - https://webdriver.io/docs/pageobjects/#get--ing-your-selectors